### ISSUE TITLE
Unable do access form from controller

### ISSUE THREAD

**Issue by renatomefidf** [vitalets/angular-xeditable#237]
Hello,

It's not possible to access the form under controller as docs say:
```html
<form editable-form name="myform">
// now myform.[property] is available in template and $scope.myform.[property] - in controller
```

I just forked you table jsfiddle: http://jsfiddle.net/renatomefidf/34tt76vw/1/
I tried a alert on save and delete, no success!

Thank you

**Comment by nicolasbousson** [comment]
I have the same problem...

**Comment by diegocortassa** [comment]
This is an old issue but, as I've lost few hours dealing with the same problem and this thread comes up on google, I just wanted to post my solution.
The problem is not really related to angular-xeditable beacuse the form get added to the scope only after the controller initialization.
The solution is to add a $watch like:
```
    $scope.$watch('myform', function(myform) {
        if(myform) {
            console.log('myform is',$scope.myform);
            $scope.myform.$show();
            $scope.myform.$visible = true;
        }
    });
```

**Comment by ckosloski** [comment]
Since there is a wontfix label on this ticket, shouldn't it just be closed?
